README.md

```diff
@@ -271,8 +294,14 @@ This sectiion contains descriptions of tables, environment variables, and jobs t
   - `instance` - the instance, like `lemmings.world`
   - `user_id` - the ID of the user
   - The only field needed is the `user_id`, but if you don't want to go looking in the db for that,
-    you can simply provide the `username` and `instance` and the automod will save the `user_id` on 
+    you can simply provide the `username` and `instance` and the automod will save the `user_id` on
     its own next time it gets any report
+- `banned_images` if an image linked in a post is similar to the image_hash, the user is banned.
+  - `image_hash` - the hash of the image to ban (more below)
```

Comment:
```suggestion
  - `image_hash` - the hash of the image to ban
```

Reply:
I've updated this with a link to the relevant section